src/calculateRentalCost.js

```diff
@@ -28,7 +28,14 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  switch (true) {
+    case days < 3 :
+      return days * 40;
+    case days >= 3 && days < 7 :
+      return days * 40 - 20;
+    case days >= 7 :
+      return days * 40 - 50;
+  }
```

Comment:
Good job!
But lets make some variables for values, because for now we don't understand what it is - 40, 50, 20 ...

Reply:
Thanks for your comment, I agree with it, so I added some variables.